### DISCUSSION TITLE
[Bug]: iOS 14 not supporting with this sdk

### DISCUSSION THREAD

**Discussion by MuraliThangavel** [oddbit/flutter_facebook_app_events#237] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

iOS 14 not supporting with this sdk and not able to trigger facebook events above iOS 14 devices

### Expected Behavior

need to work in iOS 14 devices also

### Steps To Reproduce

pubspec.yaml
flutter_facebook_auth: ^4.3.3
facebook_app_events: ^0.16.0

in Main class , 
  final facebookAppEvents = FacebookAppEvents();
  facebookAppEvents.setAdvertiserTracking(enabled: true);

### Platform

iOS

### Flutter Doctor

```shell
[✓] Flutter (Channel stable, 2.10.4, on macOS 11.6.5 20G527 darwin-x64, locale
    en-JP)
[✓] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 13.2.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.63.2)
[✓] Connected device (1 available)
[✓] HTTP Host Availability

• No issues found!
```

**Comment by banjangfriend** [comment]
I have same issue.
If you solved that, can I get some advices?

**Comment by Collins-01** [comment]
Please i am having the same issue, can you put me through?

**Comment by techFB** [comment]
For me everything seems to work, just make sure, that you've added FacebookClientToken into your Info.plist file. This token can be found in Settings > Advanced in your facebook "developer console" (or whatever they call it)

**Comment by Collins-01** [comment]
Please, what's version  of Flutter are you using?.
Android  works for me, but ios can't run

**Comment by horgag** [comment]
@Collins-01 Did you try what @techFB suggested? I ran the the project in Xcode and there was more logs. You have to add the token to your `info.plist` file.

```
 <key>FacebookClientToken</key>
		<string>ADDTOKENHERE</string>
```
